sssom/cli.py

```diff
@@ -41,6 +42,13 @@
 output_option = click.option(
     "-o", "--output", help="Output file, e.g. a SSSOM tsv file."
 )
+improved_output_option = click.option(
+    "-o",
+    "--output",
+    help="Output file, e.g. a SSSOM tsv file.",
+    type=click.File(mode="w"),
```

Comment:
note that you can have click do all of the hard work for opening files for you. below, you can even set stdout as the default. Then, later, you never have to write code that worries about file paths, handling, parsing, etc.